### PR TITLE
cloud: add aws s3 and gcs config

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Create a new article about S3 and GCS access configuration, and updated related links.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Configure Amazon S3 access and GCS access
+summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access and Google Cloud Storage (GCS) access.
+---
+
+# Configure Amazon S3 access and GCS access
+
+When you import or migrate from Amazon S3 or GCS to TiDB Cloud, you need to configure access to the Amazon S3 or GCS buckets. This document describes how to do this.
+
```
Comment (hfxsd) [inline tidb-cloud/config-s3-and-gcs-access.md:9]:
```suggestion

- [Configure Amazon S3 access](#configure-amazon-s3-access)
- [Configure Google Cloud Storage access](#configure-google-cloud-storage-access)

```

File: TOC-tidb-cloud.md [hunk 2]
```diff
@@ -37,6 +37,7 @@
     - [Import or Migrate from Amazon S3 or GCS to TiDB Cloud](/tidb-cloud/migrate-from-amazon-s3-or-gcs.md)
     - [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md)
     - [Import Apache Parquet Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-parquet-files.md)
+    - [Configure Amazon S3 Access and GCS Access](/tidb-cloud/config-s3-and-gcs-access.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:40]: Since this step is a prerequisite for migrating data, how about we make it the first doc under Migrate Data?

**Review by Frank945946 — APPROVED** [review]
LGTM

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: Configure Amazon S3 access and GCS access
+summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access and Google Cloud Storage (GCS) access.
+---
+
+# Configure Amazon S3 access and GCS access
+
+When you import or migrate from Amazon S3 or GCS to TiDB Cloud, you need to configure access to the Amazon S3 or GCS buckets. This document describes how to do this.
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md:8]:
```suggestion
If your source data is stored in Amazon S3 or GCS buckets, before importing or migrating the data to TiDB Cloud, you need to configure cross-account access to the buckets. This document describes how to do this.
```

File: TOC-tidb-cloud.md (hunk 2 above)
Comment (hfxsd) [reply TOC-tidb-cloud.md:40]: Sounds good~ Fixed.

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
+---
+title: Configure Amazon S3 access and GCS access
+summary: Learn how to configure Amazon Simple Storage Service (Amazon S3) access and Google Cloud Storage (GCS) access.
+---
+
+# Configure Amazon S3 access and GCS access
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md:6]:
```suggestion
# Configure Amazon S3 Access and GCS Access
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
+---
+title: Configure Amazon S3 access and GCS access
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md:2]:
```suggestion
title: Configure Amazon S3 Access and GCS Access
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**